samigo/samigo-app/src/webapp/jsf/qti/importAssessment.jsp

```diff
@@ -45,7 +45,7 @@
     <div class="tier1">
      <div class="tier2">
       <div class="form_label">
-       <h:messages styleClass="messageSamigo" rendered="#{! empty facesContext.maximumSeverity}" layout="table"/>
+       <h:messages styleClass="sak-banner-warn" rendered="#{! empty facesContext.maximumSeverity}" layout="table"/>
```

Comment:
You've used .sak-banner-error for others like this. Why did you use .sak-banner-warn here?

Reply:
Thanks for making this change.